interfaces/CMakeLists.txt

```diff
@@ -68,3 +68,16 @@ install(
         FILES ${JSON_HEADERS}
         DESTINATION include/${NAMESPACE}/interfaces/json
 )
+
+if(CDMI)
+        install(CODE "execute_process(  
+                COMMAND ${CMAKE_COMMAND} -E create_symlink \
+                $ENV{STAGING_DIR}/usr/include/WPEFramework/interfaces/IDRM.h \
```

Comment:
This forces STAGING_DIR to be set by every build system/environment. Could you please try to use CMAKE_INSTALL_PREFIX instead? This way it will work for most of the build systems.